app/src/main/java/de/tum/in/tumcampusapp/utils/Utils.java

```diff
@@ -96,15 +134,78 @@ public static String cutText(String text, String startString, String endString)
      * @param c          Context
      * @param key        setting name
      * @param defaultVal default value
-     * @return setting value, "" if undefined
+     * @return setting value, defaultVal if undefined
      */
     public static String getSetting(Context c, String key, String defaultVal) {
         SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(c);
         return sp.getString(key, defaultVal);
     }
 
     /**
-     * Get a value from the default shared preferences
+     * Get a value from the default shared preferences.
```

Comment:
All those shared preference accesses probably can be replaced with [Android KTX](https://github.com/android/android-ktx) in the near future...

Reply:
You still have to call `.edit` with that, but yea - it still will be some time till that reaches general availability.